pcweb/pages/docs/component_lib/forms.py

```diff
@@ -821,9 +838,6 @@ def render_upload():
             width="100%",
             padding_bottom="1em",
         ),
-        doctext(
-            "Currently we only support one file upload, but we plan to support multiple file uploads in the future. ",
-        ),
         doctext(
             "Your event handler should be an async function that accepts a single argument, ",
             pc.code("file"),
```

Comment:
we've changed the syntax of the file uploads to use lists only for uniformity reasons. This means single uploads would use multi-upload syntax(`list[UploadFile]`). In the case of single uploads, the `multiple prop` should be set to `False`.

more [here](https://github.com/pynecone-io/pynecone/pull/811)

Reply:
Thanks. Fixed.